.storybook/recipes/GlobalHeader/GlobalHeader.tsx

```diff
@@ -187,76 +186,80 @@ export const GlobalHeader = ({
         </PrimaryNav>
       </NavContainer>
       <UtilityNav className={styles['global-header__utility-nav']}>
-        <UtilityNavItem aria-label="Notifications" itemBefore={<Avatar />}>
-          <Popover
-            aria-describedby="popover-description-1"
-            aria-labelledby="popover-heading-1"
+        <Popover
+          as={React.Fragment}
+          placement={isLarge ? 'top-start' : 'bottom-end'}
```

Comment:
`isLarge` already was existing in code, and just not used, so making good use of it here